src/ray/common/ray_config_def.h

```diff
@@ -117,6 +117,11 @@ RAY_CONFIG(bool, new_scheduler_enabled,
 // Objects larger than this size will be spilled/promoted to plasma.
 RAY_CONFIG(int64_t, max_direct_call_object_size, 100 * 1024)
 
+// The max allowed size in bytes of an object returned directly in GetObjectStatus
+// replies.
+// Objects larger than this size will be put in plasma and require an object transfer.
+RAY_CONFIG(int64_t, max_get_status_object_size, 100 * 1024)
```

Comment:
We could also piggyback off of the `max_direct_call_object_size` config for now, but I thought it'd be nice to potentially let these vary separately.